### PR TITLE
turn on some caching

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -26,10 +26,10 @@ module.exports = function (karma) {
 			// These tests are commented out because the actual code doesn't get run
 			// See `src/__sw.js`
 			// 'test/caches/ads.spec.js',
-			// 'test/caches/built-assets.spec.js',
+			'test/caches/built-assets.spec.js',
 			// 'test/caches/comments.spec.js',
 			// 'test/caches/image.spec.js',
```
Comment (leggsimon) [inline karma.conf.js]: does this need un commenting as well now that you're using it?

Comment (leggsimon) [inline karma.conf.js]: And don't we have a test for the fonts caching??

Comment (wheresrhys) [inline karma.conf.js]: weird - it's a duplicate line

**Comment by wheresrhys** [comment]
good point. Thought they already were.

**Pull request closed without merging**